src/components/oscal-utils/OSCALUtils.js

```diff
@@ -0,0 +1,62 @@
+export function deepClone(obj) {
+  return JSON.parse(JSON.stringify(obj));
+}
+
+/**
+ *
```

Comment:
Function name is fairly descriptive, but it might be nice to have a brief explanation on the function in the doc. Especially given the params are nicely described.